regolith/builders/recentcollabsbuilder.py

```diff
@@ -47,7 +47,10 @@ def construct_global_ctx(self):
     def latex(self):
         rc = self.rc
         since_date = dt.date.today() - relativedelta(months=48)
-        person = fuzzy_retrieval(all_docs_from_collection(rc.client, "people"), ['aka', 'name', '_id'], self.rc.people, case_sensitive = False)
+        target = self.rc.people if self.rc.people else "sbillinge"
```

Comment:
Yes, I caught this error too.  This is a good fix but let's not hard-code "sbillinge" in here.  Call it DEFAULT_PERSON or sth and assign it at the top of the module.